explore/src/main/scala/explore/proposal/ProposalEditor.scala

```diff
@@ -611,6 +613,25 @@ object ProposalEditor:
             else
               ps.set(cfpPartners.map(p => PartnerSplit(p, 0.refined)))
           )
+      // Update the deadline when the call or partner splits change
+      .useEffectWithDepsBy((props, _, cfps, _, _, splits) =>
+        (props.proposal.get.callId, cfps.toOption.orEmpty, splits.get)
+      ): (props, _, _, _, _, ps) =>
+        (callId, cfps, splits) =>
+          callId.fold(Callback.empty)(cid =>
```

Comment:
Doesn't `foldMap` work with `Callback`? Otherwise `.map(..).getOrEmpty` should work.

Reply:
Good question, I'll check